src/modules/json-ld/components/BreadcrumbsJsonLd.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import BreadcrumbModel from '../../common/BreadcrumbModel'
+
+const createJsonLd = (breadcrumbs: Array<BreadcrumbModel>) => {
+  // https://developers.google.com/search/docs/data-types/breadcrumb
+  return {
```

Comment:
Did you check the format using this tool? https://search.google.com/structured-data/testing-tool?hl=de

Reply:
He did :clap: